packages/api-sdk/src/medical/models/patient.ts

```diff
@@ -24,7 +25,13 @@ export const patientListSchema = z.object({
 export const queryStatusSchema = z.enum(["processing", "completed", "failed"]);
 export type QueryStatus = z.infer<typeof queryStatusSchema>;
 
-export type QueryProgress = {
-  status: QueryStatus | null;
+export type GetConsolidatedQueryProgressResponse = {
+  status: QueryStatus;
```

Comment:
Can we add the `@deprecated` TSDoc to this prop here as well, please? When we add it to the type, anywhere we hover over that property we'll get this notification.